### DISCUSSION TITLE
Got in fastboot mode OPPO without RSA key

### DISCUSSION THREAD

**Discussion by NasiGorengBurung** [zenfyrdev/bootloader-unlock-wall-of-shame#16] [Show and tell]
So. I managed to get in fastboot mode in my OPPO Reno7 5G. It say "Fastbootd mode" with mispelling. It can be achieve by adb 'adb reboot fastboot"

But, fastboot mode look same as recovery mode as normally by vol down + power

**Comment by zenfyrdev** [comment]
This is not a misspelling, this is FastbootD, which is userspace fastboot. https://source.android.com/docs/core/architecture/bootloader/fastbootd
It's basically useless without an unlocked bootloader, which requires fastboot/bootloader access. Otherwise, it's used to manipulate dynamic partitions inside `super`. (Flash GSIs)

**Reply by NasiGorengBurung** [reply]
Oh then. Thanks for informing that

**Comment by amartinezdiaz747-ship-it** [comment]
I need help with an Oppo A11 model PCHM10. It gets stuck on startup and there's a tool other than the MSM download tool for obvious reasons. Can someone please help me?
